Kudu.Core/Deployment/Generator/WebSiteBuilder.cs

```diff
@@ -5,32 +5,31 @@
 
 namespace Kudu.Core.Deployment.Generator
 {
-    public class WebSiteBuilder : GeneratorSiteBuilder
+    public class WebSiteBuilder : MicrosoftSiteBuilder
     {
         private readonly string _projectPath;
-        private readonly string _solutionPath;
-
         public WebSiteBuilder(IEnvironment environment, IDeploymentSettingsManager settings, IBuildPropertyProvider propertyProvider, string sourcePath, string projectPath, string solutionPath)
-            : base(environment, settings, propertyProvider, sourcePath)
+            : base(environment, settings, propertyProvider, sourcePath, null, solutionPath, "--aspWebSite")
         {
-            _projectPath = projectPath;
-            _solutionPath = solutionPath;
+            _projectPath = CleanPath(projectPath);
         }
 
+        // different from other MSSiteBuilders which does this "--aspWebSite {projectFilePath}",
+        // instead it uses "--aspWebSite --sitePath {projectPath}"
```

Comment:
nvm...website is a special case, it doesn't take `projectFilePath` but it has `sitePath`